src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -57,13 +72,7 @@ public function setDebug($debug)
      */
     public function getUser()
     {
-        if (null !== $this->tokenStorage) {
```

Comment:
not sure this change is needed, and it forces the error message to be less specific